plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestConfluentSchemaRegistryTableDescriptionSupplier.java

```diff
@@ -133,6 +159,27 @@ public void testOverriddenSubject()
                                         ImmutableList.of(getFieldDescription("overridden_col1", INTEGER), getFieldDescription("overridden_col2", createUnboundedVarcharType()))))));
     }
 
+    @Test
+    public void testAmbiguousOverriddenSubject()
+            throws Exception
+    {
+        TableDescriptionSupplier tableDescriptionSupplier = createTableDescriptionSupplier();
+        String topicName = "base_Topic";
+        SchemaTableName schemaTableName = new SchemaTableName(DEFAULT_NAME, topicName);
+        String overriddenSubject = "ambiguousOverriddenSubject";
+        SCHEMA_REGISTRY_CLIENT.register(overriddenSubject, getAvroSchema(schemaTableName.getTableName(), "overridden_"));
+        SCHEMA_REGISTRY_CLIENT.register(overriddenSubject.toUpperCase(ENGLISH), getAvroSchema(schemaTableName.getTableName(), "overridden_"));
+
+        assertThatThrownBy(() ->
+                tableDescriptionSupplier.getTopicDescription(
+                        TestingConnectorSession.builder()
+                                .setPropertyMetadata(new ConfluentSessionProperties(new ConfluentSchemaRegistryConfig()).getSessionProperties())
+                                .build(),
+                        new SchemaTableName(DEFAULT_NAME, format("%s&value-subject=%s", topicName, overriddenSubject))))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Subject 'ambiguousoverriddensubject' is ambiguous, and may refer to one of the following: ambiguousOverriddenSubject, AMBIGUOUSOVERRIDDENSUBJECT");
```

Comment:
Table is ambigous?

Reply:
I think it should be `subject` since we are resolving subject names for key and value which are extracted from the table name